app/views/languages/language.erb

```diff
@@ -48,7 +55,38 @@
 
     <div class="col-md-8 tab-content">
 
-      <div class="active tab-pane" id="about">
+      <% if !track.active %>
+        <div class="active tab-pane" id="launch">
+          <h5>Help Us Launch the <%= track.language %> Track!</h5>
+
+          <p>
+            Launching a track takes a lot of work and we'd love your help!
+          </p>
+
+          <h5> In order to launch we need: </h5>
+
+          <p>
+            <ul>
+              <li>A submodule in x-api</li>
+              <li>At least 10 problems</li>
+              <li>A "how to get started" topic in the help repo (app/pages/languages/getting-started-with-groovy.md)</li>
```

Comment:
Take a look at the checklist at https://github.com/exercism/xelm/issues/1 -- we've changed how the documentation is handled, it's now in `docs/` inside each language repo.